src/blender-injection/extern.cpp

```diff
@@ -23,32 +23,65 @@ const std::vector<std::string> SUPPORTED_FORMATS = {
 
 std::vector<std::intptr_t> isAlreadyTriggered = {};
 
+const char* GetDropFilePath(void* pUnknown)
+{
+    if (BlenderPatcher::GetInstance()->GetVersion() >= std::make_tuple(3, 6, 0))
```

Comment:
Blender 3.6.0 (or greater) does not have `path` property on `wmDrag`, but `poin` represents `wmDragPath` and it has `path` property.